itou/www/invitations_views/forms.py

```diff
@@ -143,6 +143,11 @@ def __init__(self, invitation, *args, **kwargs):
         self.fields["first_name"].initial = invitation.first_name
         self.fields["last_name"].initial = invitation.last_name
 
+    def clean(self):
```

Comment:
Attention, cette méthode est commune à tous les types d'invitation. Or certaines invitations n'ont pas de siae (c'est le cas des prescripteurs).
Tu peux peut-être faire ça : 
```python
if getattr(self.invitation, "siae"):
  if not self.invitation.siae.is_active:
    ...
```

Reply:
Note to self `if model_string == cls.GUEST_TYPE_SIAE_STAFF`

Reply:
Je me demandais pourquoi cela n'avait pas cassé sur un test d'invitation prescripteur, mais forcément il est dans ta PR non mergée :-D

Reply:
`isinstance(self.invitation, SiaeStaffInvitation)`